### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by karinamzalez**
- fix styling 
- make high scores functional 
- limit high scores to top 5 
- all 60 tests passing

### CODE REVIEW

**Comment by chompasina** [comment]
Looks good!

File: index.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h3 class="panel-title">Instructions</h3>
   <!--/container--></div>
 
 
-<a href="http://www.freepik.com/free-photos-vectors/food">Food vector designed by Bakar015 - Freepik.com</a>
+<a href="http://www.freepik.com/free-photos-vectors/food"></a>
```
Comment (chompasina) [inline index.html]: I think we need this to be in there.

File: lib/index.js [hunk 2]
```diff
@@ -103,7 +109,7 @@ function endGame(){
 function addScore(winningScore){
   if(scoreBoard.scores[0] && trayCollision.total > Math.min.apply(Math, scoreBoard.scores)){
     scoreBoard.addHighScoreToPage(winningScore);
-  } else if (scoreBoard.length === 0){
+  } else {
     scoreBoard.addHighScoreToPage(winningScore);
```
Comment (chompasina) [inline lib/index.js:113]: Good refactor to catch all other cases

File: index.html (hunk 1 above)
Comment (karinamzalez) [reply index.html]: ok!

File: lib/score-board.js [hunk 3]
```diff
@@ -1,31 +1,37 @@
...
+    scoreItem.innerText = score;
+    list.appendChild(scoreItem);
+    scores.push(score);
   });
-  }
 };
 
+function uniq(a) {
+    return a.sort().filter(function(item, pos, ary) {
+        return !pos || item !== ary[pos - 1];
```
Comment (chompasina) [inline lib/score-board.js:24]: I know this function is pretty self-explanatory but more verbose variable names might make it even easier to understand... what do you think?

**Pull request merged**